### DISCUSSION TITLE
colorRange

### DISCUSSION THREAD

**Discussion by SandraMarq** [open-sdg/open-sdg#2000] [Q&A] [answered]
Hello

Can you help me implement this part of the map? please

![image](https://github.com/open-sdg/open-sdg/assets/118024145/d40c9a13-657e-4554-a104-9f6b7c904a76)

**Comment by brockfanning** [comment]
@SandraMarq With the new options there is quite a bit of flexibility. What are your requirements for the map colors?

**Reply by SandraMarq** [reply]
Hello

I want to know in which file the function myMapColorFunction is put.

Thank you

**Reply by brockfanning** [reply, marked as answer]
@SandraMarq If you started with the "site starter" then you should have a custom.js file here: https://github.com/open-sdg/open-sdg-site-starter/blob/develop/assets/js/custom.js

That might be the easiest place to add the function.

However as long as it is in any JavaScript file that is loaded on the page, that should work. So if you have JavaScript in some other file that is probably fine too.

**Reply by SandraMarq** [reply]
Thanks for your help.
